src/components/Header.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+export function Header(): JSX.Element {
+  return (
+    <header className="p-3 mb-3 align-right" style={{ background: '#009cab' }}>
+      <div className="container">
+        <HeaderLink to="/">
+          <HeaderLogo />
+        </HeaderLink>
+
+        <nav className="float-right">
+          <HeaderLink to="/">Tracks</HeaderLink>
+          <HeaderLink to="/stories">Stories</HeaderLink>
+        </nav>
+      </div>
+    </header>
+  )
+}
+
+function HeaderLogo(): JSX.Element {
+  return (
+    <img
+      alt="Exercism"
```

Comment:
```suggestion
      alt="Exercism logo"
```

Will be announced as "Exercism logo image"